### ISSUE TITLE
Xanthine dehydrogenase, small subunit (IPR014307)

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-annotation#4157]
* **InterPro ID / label**
Xanthine dehydrogenase, small subunit (IPR014307)

* **Example sequences with problematic annotation (ID + gene/protein name):**
fly rosy (FBgn0003308/P10351)

* **Description of issue**
This entry currently has two GO MF enzymatic terms associated:
xanthine dehydrogenase activity (GO:0004854)
xanthine oxidase activity (GO:0004855)

But I think it should only have the first one (NAD-dependent xanthine dehydrogenase activity).  What's the evidence for including the second one too?

**Comment by sarach06** [comment]
Hi Steven,
Thank you very much for pointing this out. It's been described that some mammalian xanthine dehydrogenases can be converted to XOD post-translationally (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5094624/) and I think that's why this term was added. I removed the term as it doesn't apply for all members, i.e chicken XDH can't be converted as stated in the article I mentioned. Does it make sense?
Thanks, 
Sara
